test/fixtures/dialects/sparksql/select_lateral_view_supported_tvf.yml

```diff
@@ -96,7 +96,7 @@ file:
                           numeric_literal: '4'
                       - end_bracket: )
                   end_bracket: )
-            - keyword: AS
+            - naked_identifier: AS
```

Comment:
I think this is a regression. What's the reason for changing AS to unreserved?

Reply:
I noticed that `AS` wasn't a reserved keyword in SparkSQL default mode so I updated this. Additionally, I think the reason we are seeing `naked_identifier` here is moreso due to the structure of the Lateral View clause, I updated it to use the Alias Expression Segment (since `AS` is now optional) and we see the keyword has been restored.